README.md

```diff
@@ -1,2 +1,184 @@
-# aws-eden-cli
-ECS Dynamic Environment Manager (eden) CLI
+# aws-eden-cli: ECS Dynamic Environment Manager [![PyPi version](https://img.shields.io/pypi/v/aws-eden-cli.svg)](https://pypi.python.org/pypi/aws-eden-cli/) ![](https://img.shields.io/badge/python-3.6+-blue.svg) ![t](https://img.shields.io/badge/status-beta-orange.svg) 
+
+
+Clone ECS environments easily. 
+Provide eden with a sample ECS service and eden will clone it. 
+
+eden is provided in CLI and Terraform module (Lambda with HTTP API) flavors. 
+You can use HTTP API from CI of your choice on Pull Request open/close, 
+new commit pushes to fully automate environment creation. 
+For API flavor, see eden API at [GitHub](https://github.com/baikonur-oss/terraform-aws-lambda-eden-api).
+
+Works only with Python 3.6+.
+
+## Developing with eden
+
+![simple-figure](figures/aws-eden-simple-en.png)
+
+## Requirements:
+1. Config JSON file in a S3 bucket with structure described below
+2. A reference ECS Service with Target Group Attached
+3. An ALB with HTTPS Listener
```

Comment:
Don't we need a wildcard certificate for host header based routing?

Reply:
We sure do!